bin/envconfig

```diff
@@ -114,7 +114,11 @@ if $INTERACTIVE ; then
     envs[l]=$(echo "${i}" | cut -f 1 -d ' ')
     echo "${l} - ${envs[${l}]}"
   done
+  echo "0 EXIT (return)"
   read -r -p "Choose environment [1 to ${l}]: " eid
```

Comment:
Maybe include the `0 to exit` in the prompt?

Reply:
0 is not an environment. But if you feel it is better to include it, I do.

Reply:
I mean a prompt something like:
```
Choose environment [1 to ${l}; 0 to exit]: 
```